pkgs/development/python-modules/easysnmp/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, lib
+, pkgs
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+
+, python
+, pytest
+, pytestcov
+, pytest-flake8
+, pytest-sugar
+, termcolor
+, ipython
+}:
+
+buildPythonPackage rec {
+  pname = "easysnmp";
+  version = "0.2.5";
+
+  # See https://github.com/kamakazikamikaze/easysnmp/issues/108
+  disabled = !(pythonOlder "3.7");
+
+  src = fetchFromGitHub {
+    owner = "kamakazikamikaze";
+    repo = pname;
+    rev = version;
+    sha256 = "1si9iyxqj6z22jzn6m93lwpinsqn20lix2py3jm3g3fmwawkd735";
+  };
+
+  checkInputs = [
+    pytest
+    pytestcov
+    pytest-flake8
+    pytest-sugar
+    termcolor
+    pkgs.net_snmp
+  ];
+
+  buildInputs = [
+    pkgs.net_snmp
+    pkgs.openssl
+    pkgs.gcc
+    ipython
+  ];
+
+  patches = [./easysnmp.patch];
+
+  buildPhase = ''
+    python setup.py build --basedir=${pkgs.net_snmp}/bin
+  '';
+
+  installPhase= ''
```

Comment:
This installPhase largely does what the default buildPythonPackage installPhase does, you should just remove this altogether.

Reply:
Doing so causes the following:
```
/nix/store/f74zj9535vgk9nlrskk1qgk3v6m7w6n4-stdenv-linux/setup: line 1324: pushd: dist: No such file or directory
```

Reply:
it's because you're overriding the buildPhase, buildPhase normall builds a wheel which would be `python setup.py build bdist_wheel`

this worked:
```

  buildPhase = ''
    python setup.py build bdist_wheel --basedir=${net_snmp}/bin
  '';

  # Unable to get tests to pass, even running by hand. The pytest tests have
  # become stale.
  doCheck = false;

```

Reply:
Ah yes, that does work!  Thank you.